src/backend/web/graphql/index.js

```diff
@@ -26,15 +26,14 @@ module.exports.typeDefs = graphql`
   # 'Post' matches our Post type used with redis
   type Post {
     id: String
-    author: String
     title: String
     html: String
     text: String
     published: String
     updated: String
     url: String
-    site: String
     guid: String
+    feed: String
   }
```

Comment:
We can make feed return a type [Feed] if we go with the latter option (I think). The problem is I don't know if you can do nested queries both ways. When I played with nested queries, I changed the Feed schema so that it also has posts like below: 
![image](https://user-images.githubusercontent.com/18711727/74475723-f4988d80-4e75-11ea-8346-9119bca09a16.png)

So when you get Feed, you can see posts by that author.

What I'm not sure of is if you can do that for Feed AND change the schema for Post so that it does the following at the same time.
```
 type Post {
    id: String
    title: String
    html: String
    text: String
    published: String
    updated: String
    url: String
    guid: String
    feed: [Feed]
  }
```